airbyte-webapp/src/packages/cloud/services/thirdParty/fullstory/useFullStory.tsx

```diff
@@ -18,6 +20,23 @@ const useFullStory = (
     }
   }, [config]);
 
+  useEffect(() => {
+    if (enabled) {
+      if (user) {
+        // We have a user, so identify that user information against fullstory
+        FullStory.identify(user.userId, {
+          email: user.email,
+          displayName: user.name,
+          intercomHash: user.intercomHash,
```

Comment:
Are we sure that we want to send intercomHash to fullstory?

Reply:
Yes, that helps actually filtering down in Fullstory, once you have the intercom hash (since support will also be using fullstory more).